src/BaGet.UI/src/config.tsx

```diff
@@ -1,13 +1,22 @@
 let config = {
-  apiUrl: "__BAGET_PLACEHOLDER_API_URL__"
+  apiUrl: "__BAGET_PLACEHOLDER_API_URL__",
+  baseUrl: "__BAGET_PATH_BASE_PLACEHOLDER__"
```

Comment:
This should start with `__BAGET__PLACEHOLDER_`, otherwise line 10 doesn't work.

```suggestion
  baseUrl: "__BAGET_PLACEHOLDER_PATH_BASE__"
```